src/components/MutationsTable.tsx

```diff
@@ -0,0 +1,271 @@
+};
+const filterBaselineMutations = (mutations: MutationProportionEntry[]) => {
+  return mutations.filter(mutation => mutation.proportion > 0.9);
+};
+
+const columnNames: string[] = mutationColumnsProperties.map(item => item.label);
+
+export const SingleMutationsTable = ({
+  mutations,
+  locationSelector,
+  mutationType,
+  dateRangeSelector,
+}: SingleMutationsTableProps) => {
+  return (
+    <>
+      <div className='table-container'>
+        <table className='mutationCustomTable' data-testid='single-mutation-table'>
+          <thead>
+            <tr>
+              <td>Name</td>
+              {columnNames.map(name => (
+                <td key={name}>{name}</td>
+              ))}
+            </tr>
+          </thead>
+          <tbody>
+            {mutations.map(item => (
+              <tr key={item.name}>
+                <td>
+                  <NameCell
+                    query={item.query}
+                    locationSelector={locationSelector}
+                    name={item.name}
+                    baseline={false}
+                    dateRangeSelector={dateRangeSelector}
+                  />
+                </td>
+                {mutationColumnsProperties.map(field => {
+                  return <td key={field.label}>{displayMutations(item, mutationType, field)}</td>;
+                })}
+              </tr>
+            ))}
+          </tbody>
+        </table>
+      </div>
+    </>
+  );
+};
+
+export type BaselineMutationsTableProps = {
+  baselineData: BaselineData;
+  mutations: MutationObject[] | undefined;
+  mutationType: SequenceType;
+  locationSelector: LocationSelector;
+  baselineVariant?: VariantSelector;
+  dateRangeSelector: DateRangeSelector;
+};
+
+export const BaselineMutationsTable = ({
+  baselineData,
+  mutations,
+  mutationType,
+  locationSelector,
+  baselineVariant,
+  dateRangeSelector,
+}: BaselineMutationsTableProps) => {
+  const headers: string[] = ['Additional', 'Missing'];
+  console.log('baseline data', baselineData);
+
+  return (
+    <>
+      <div className='mb-4 italic'>
+        *The table compares the mutation set of the variants in the collection to the fixed /{'>'}90%
+        mutations of the baseline.
+      </div>
+      <div className='table-container'>
+        <table className='mutationCustomTable' data-testid='baseline-mutation-table'>
+          <thead>
+            <tr>
+              <td> </td>
+              {headers.map(header => (
+                <td key={header} colSpan={4}>
+                  {header}
+                </td>
+              ))}
+            </tr>
+            <tr>
+              <td>Name</td>
+              {columnNames.map((name, i) => (
+                <td key={`${name}-${i}`}>{name}</td>
+              ))}
+              {columnNames.map((name, i) => (
+                <td key={`${name}/${i}`}>{name}</td>
+              ))}
+            </tr>
+          </thead>
+          <tbody>
+            {mutations &&
```

Comment:
Same here: Please require in the Props that mutations is not undefined and remove this check here.